### PR TITLE
#7133 feat: Gallery thumbnail view - prevent orphans

### PR DESCRIPTION
**Pull request by creido-welly**
This PR adds logic to insert a row break which will force the 2nd to last gallery item on to the next row to prevent any orphans

### CODE REVIEW

File: src/components/Gallery/Gallery.tsx [hunk 1]
```diff
@@ -103,7 +106,7 @@ export const GalleryMedia = ({
           />
         </Button>
       </div>
-      {isLead && <div className="break" />}
```
Comment (stuartwilkes) [inline src/components/Gallery/Gallery.tsx:106]: Just checking - is this change for style consistency, or a technical reason? Just wondering!

**Review by stuartwilkes — APPROVED** [review]

**Comment by lukewhitmore** [comment]
Just out of interest, could this have been carried out purely using flexbox? What are the benefits of using the extra element?

File: src/components/Gallery/Gallery.tsx (hunk 1 above)
Comment (creido-welly) [reply src/components/Gallery/Gallery.tsx:106]: It is (unfortunately) purely for layout/styling reasons. Loathe as I am to use non-semantic styles and coupled CSS+HTML, @chriddell and I talked thru options at the time and realised that this was really the only way to make row breaks work

**Comment by creido-welly** [comment]
> Just out of interest, could this have been carried out purely using flexbox? What are the benefits of using the extra element?

@lukewhitmore unfortunately not (I tried 😢 ) https://github.com/wellcometrust/corporate-components/pull/246#discussion_r468453832

File: src/components/Gallery/Gallery.tsx (hunk 1 above)
Comment (stuartwilkes) [reply src/components/Gallery/Gallery.tsx:106]: Fair enough - there's always cases where the non-idealistic solution is the best option...

**Pull request merged**